bin/usagi

```diff
@@ -8,8 +8,33 @@ require 'usagi'
 # > --usagi-allow-store-key-reuse:  Allows to use STORE_VALUE multiple times with same key name
 # > --usagi-allow_nil_store_values: Allows to store nil value for later use with REUSE_VALUE
 
+if ARGV.grep(/^-(h|-help)$/).length > 0
+  puts <<-EOHELP
+    Usagi is a functional test suite for rails APIs based on RSpec.
+
+    All arguments not starting with `--usagi` will be passed to RSpec.
+
+    Arguments:
+      --help, -h
+        Displays this help message
+      --usagi-debug-requests
+        Outputs all curl payloads as REQ and the results data as RES.
+      --usagi-rails-output
+        Shows buffered rails for all requests.
+      --usagi-allow-store-key-reuse
+        Allows to use the STORE_VALUE directive multiple times with the
+        same key name.
+      --usagi-allow-nil-store-values
+        Allows to store nil values for later use with REUSE_VALUE.
+  EOHELP
+  Process.exit(0)
+end
+
 usagi_args = ARGV.grep(/--usagi-/)
-Usagi.start(*usagi_args)
+
+Dir.chdir(ENV['USAGI_APP_ROOT'] || "./") do
```

Comment:
Mais quel gros pd. Je suis surtout CTO pour s'assurer qu'on S(ave)hip it fast et qu'on keep it simple. Gérer la présence d'une variable d'env m'a paru moins overkill qu'une gestion d'arguments mais je ne sais pas a quel point et sur quels plans ça affecte votre Gem.

**Faut dire que le README est tellement bien détaillé...**

![screen shot 2016-01-20 at 10 54 55](https://cloud.githubusercontent.com/assets/513737/12445402/55e57e5e-bf64-11e5-8f13-85a9e28c4b32.png)

Hannnnnnn!!

Love. <3